nym-node-status-api/src/monitor/mod.rs

```diff
@@ -97,14 +113,16 @@ async fn run(
         .log_error("get_cached_mixnodes")?;
     tracing::debug!("Fetched {} mixnodes", mixnodes.len());
 
-    // TODO dz can we calculate blacklisted GWs from their performance?
-    // where do we get their performance?
-    let gateways_blacklisted = api_client
-        .nym_api
-        .get_gateways_blacklisted()
-        .await
-        .map(|vec| vec.into_iter().collect::<HashSet<_>>())
-        .log_error("get_gateways_blacklisted")?;
+    let gateways_blacklisted = skimmed_gateways
```

Comment:
remind me again, why do we still care about blacklist?

Reply:
It's part of reporing on the gateways page 

![image](https://github.com/user-attachments/assets/5cf4f6af-8348-4425-a510-38089c50708d)


and part of overall network summary stats

Reply:
gotcha. hopefully we'll be able to get away from it at some point...